buildSrc/src/main/groovy/airbyte-docker.gradle

```diff
@@ -56,10 +56,30 @@ abstract class AirbyteDockerTask extends DefaultTask {
         }
     }
 
+    def buildDockerfileWithLocalCdk(String scriptPath, String fileName) {
+        if (project.file(fileName).exists()) {
+            def tag = DockerHelpers.getDevTaggedImage(projectDir, dockerfileName)
+            project.exec {
+                environment "CONNECTOR_TAG", tag
+                environment "CONNECTOR_NAME", project.findProperty('connectorAcceptanceTest.connectorName')
+                commandLine scriptPath
+            }
+        }
+    }
+
     @TaskAction
     def dockerTask() {
-        def scriptPath = Paths.get(rootDir.absolutePath, 'tools/bin/build_image.sh').toString()
-        buildDockerfile(scriptPath, dockerfileName)
+        if (
+            project.hasProperty('connectorAcceptanceTest.useLocalCdk') &&
```

Comment:
why have these two separate checks? is the second just checking if the value is truth-y?

Reply:
Yes the second checks to see if the value is truthy.

I've tried using `getProperty` here instead of the combination of `hasProperty` + indexing but get an exception in the [Octavia Cli: Build step](https://github.com/airbytehq/airbyte/actions/runs/4292402149/jobs/7478793758): 
```
Execution failed for task ':octavia-cli:airbyteDocker'.
> Could not get unknown property 'connectorAcceptanceTest.useLocalCdk' for project ':octavia-cli' of type org.gradle.api.Project.
```